.github/workflows/l10n_extract.yaml

```diff
@@ -0,0 +1,64 @@
+name: Extract Strings
+on:
+  schedule:
+    - cron: '0 22 * * 4' # At 15:00 PDT on Thursday
+  workflow_dispatch:
+jobs:
+  extract:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Linux packages
+        run: |
+          sudo apt update
+          sudo apt install gettext graphicsmagick git hub translate-toolkit -y
```

Comment:
Couple of notes here.

I'm using `hub`. The previous workflow was using the [gh npm package](https://www.npmjs.com/package/gh) which looks deprecated.

I'm also installing `translate-toolkit`. `merge_po.sh` has a conditional on `podebug` being available to generate `db_LB`, not sure if we want to skip this part.

Reply:
Actually, it looks like this is failing because the locale doesn't exist. Given we're moving to Fluent, I guess we can ignore it (as we were doing in Circle CI)

```
podebug: WARNING: Error processing: input locale/templates/LC_MESSAGES/messages.pot, output locale/db_LB/LC_MESSAGES/messages.po, template None: [Errno 2] No such file or directory: 'locale/templates/LC_MESSAGES/messages.pot'
```

Reply:
Confirmed, it's not needed.